operator/controllers/btpoperator_controller.go

```diff
@@ -859,10 +873,21 @@ func (r *BtpOperatorReconciler) HandleReadyState(ctx context.Context, cr *v1alph
 	ready, err := manifest.ConsistencyCheck(&logger, installInfo, []types.ObjectTransform{r.labelTransform})
 	if err != nil {
 		logger.Error(err, fmt.Sprintf("error while checking consistency of resource %s", client.ObjectKeyFromObject(cr)))
-		return r.UpdateBtpOperatorState(ctx, cr, types.StateError)
+		return r.UpdateBtpOperatorStatus(ctx, cr, types.StateError, ConsistencyCheckFailed, fmt.Sprintf("error while checking consistency of resource %s", client.ObjectKeyFromObject(cr)))
 	} else if !ready {
-		return r.UpdateBtpOperatorState(ctx, cr, types.StateProcessing)
+		return r.UpdateBtpOperatorStatusAndLogStateChange(ctx, cr, types.StateProcessing, Initialized, "Consistency checked")
```

Comment:
I suggest to change the message to "Chart is inconsistent. Reconciliation initialized"

Reply:
In the next commits.